common/djangoapps/track/shim.py

```diff
@@ -101,6 +103,10 @@ def __call__(self, event):
         if name not in NAME_TO_EVENT_TYPE_MAP:
             return
 
+        #Convert edx.video.seeked to edx.video.positiion.changed
```

Comment:
Also, comments should have a space after the '#'.  I'm surprised that wasn't complained about by pep8/pylint.